fast64_internal/sm64/sm64_anim.py

```diff
@@ -176,12 +176,41 @@ def exportAnimationC(armatureObj, loopAnim, dirPath, dirName, groupName,
 
 	# write to table.inc.c
 	tableFilePath = os.path.join(animDirPath, 'table.inc.c')
+
+	# if table doesn´t exist, create one
 	if not os.path.exists(tableFilePath):
 		tableFile = open(tableFilePath, 'w', newline='\n')
 		tableFile.write('const struct Animation *const ' + \
 			animsName + '[] = {\n\tNULL,\n};\n')
 		tableFile.close()
-	writeIfNotFound(tableFilePath, '\t&' + sm64_anim.header.name + ',\n', '\tNULL,\n};')
+		
+	fileData = open(tableFilePath, 'r')
+	fileData.seek(0)
+	stringData = fileData.read()
+	fileData.close()
```

Comment:
```suggestion
	with open(tableFilePath, 'r') as f:
		stringData = f.read()
```
(if you want to read, see
https://docs.python.org/3/tutorial/inputoutput.html#tut-files
https://docs.python.org/3/reference/datamodel.html#context-managers )

Reply:
you're a lot less late than we are on the review :sweat_smile: , and to me not late at all anyway, few days of delay is perfectly fine